libraries/GCS_MAVLink/GCS.cpp

```diff
@@ -193,6 +195,16 @@ void GCS::update_sensor_status_flags()
         }
     }
 
+    // give GCS status of prearm checks. This is enabled if any arming checks are enabled.
```

Comment:
not a good one for backport

Reply:
Already approved above and at least 4 users asked for this.